modules/installation-infrastructure-user-infra.adoc

```diff
@@ -44,19 +45,48 @@ ifdef::ibm-z[]
 . Set up an HTTP or HTTPS server to provide Ignition files to the cluster nodes.
 endif::ibm-z[]
 ifndef::ibm-z[]
-. Configure DHCP or set static IP addresses on each node.
+. If you are using DHCP to provide the IP networking configuration to your cluster nodes, configure your DHCP service.
+.. Add persistent IP addresses for the nodes to your DHCP server configuration. In your configuration, match the MAC address of the relevant network interface to the intended IP address for each node.
+.. When you use DHCP to configure IP addressing for the cluster machines, the machines also obtain the DNS server information through DHCP. Define the persistent DNS server address that is used by the cluster nodes through your DHCP server configuration.
++
+[NOTE]
+====
+If you are not using a DHCP service, you must provide the IP networking configuration and the address of the DNS server to the nodes at {op-system} install time. These can be passed as boot arguments if you are installing from an ISO image. See the _Creating Red Hat Enterprise Linux CoreOS (RHCOS) machines_ section for more information about static IP provisioning and advanced networking options.
+====
++
+.. Define the host names of your cluster nodes in your DHCP server configuration. See the _Setting the cluster node host names through DHCP_ section for details about host name considerations.
```

Comment:
s/host names/hostnames/

Reply:
Ha, I did the same thing! (Started to suggest spelling as one word but realized most occurrences are already there, so didn't bring it up.)

Reply:
Thank you for highlighting this. I have changed this to hostname/hostnames throughout the PR.